wget_retry.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+RETRIES=10
+SLEEPTIME=5
+COUNT=0
+while [ $COUNT -lt $RETRIES ]
+do
+    wget --progress=dot:giga --retry-connrefused --waitretry=5 --read-timeout=20 --timeout=15 --tries 20 --continue "$@"
```

Comment:
The reason this script exists is because wget wont retry on 404, hence the exit status check.

I had a look at the man pages on my local machine and the option you mention isnt mentioned.